### PR TITLE
fix(docs): fixed not found error

### PR DESCRIPTION
**Pull request by MadeGOD**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested.

**Semantic versioning classification:**

- [ ] This PR includes new feature, methods or parameters
  - [ ] This PR includes breaking changes (feature, methods, parameters removed or renamed)
- [ ] This PR **only** includes non-code(typo, documentation etc.) changes

### CODE REVIEW

File: src/utils/djsDocs.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const Discord = require('discord.js')
 
 module.exports.source = version => {
   const parsed = version.match(/^(\d+)\.\d+\.\d+$/)?.[1]
-  return `https://raw.githubusercontent.com/discordjs/discord.js/docs/${parsed ? `v${parsed}` : 'main'}.json`
+  return `https://raw.githubusercontent.com/discordjs/discord.js/docs/${parsed ? `v${parsed}` : 'stable'}.json`
```
Comment (wonderlandpark) [inline src/utils/djsDocs.js:6]: Why are you changing this?

File: src/utils/djsDocs.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ module.exports.docs = async (q) => {
   const params = new URLSearchParams({ force: true, src: this.source(Discord.version), q })
   const res = await fetch(`https://djsdocs.sorta.moe/v2/embed?${params}`)
   const embed = await res.json()
-  console.log(embed)
-  if (!embed) return 'Nothing found'
+  //console.log(embed)
```
Comment (wonderlandpark) [inline src/utils/djsDocs.js:17]: DO NOT CHANGE CODE THAT IS NOT RELATED TO PR

**Review by wonderlandpark — CHANGES_REQUESTED** [review]

File: src/utils/djsDocs.js [hunk 3]
```diff
@@ -3,6 +3,8 @@ const Discord = require('discord.js')
 
 module.exports.source = version => {
   const parsed = version.match(/^(\d+)\.\d+\.\d+$/)?.[1]
+  if(parsed === '13')
```
Comment (MadeGOD) [inline src/utils/djsDocs.js:6]: If the version is 13, it is made to move to stable.

**Review by MadeGOD — COMMENTED** [review]
@wonderlandpark

**Comment by wonderlandpark** [comment]
I commented YOU SHOULD NOT CHANGE ANY OF THOSE CODES THAT ARE NOT RELATED TO PR.

You must create the least amount of PR. That so, closing this PR.
